portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <v-navigation-drawer
+    v-model="connectionsDrawer"
+    absolute
+    right
+    stateless
+    temporary
+    width="420"
+    max-width="100vw"
+    max-height="100vh"
```

Comment:
Could you avoid putting styles inside the DOM and use CSS classes.

Reply:
Okay